packages/client/hmi-client/src/components/workflow/scenario-templates/horizon-scanning/horizon-scanning-scenario.ts

```diff
@@ -0,0 +1,334 @@
+import { BaseScenario } from '@/components/workflow/scenario-templates/base-scenario';
+import * as workflowService from '@/services/workflow';
+import { operation as ModelOp } from '@/components/workflow/ops/model/mod';
+import { operation as ModelConfigOp } from '@/components/workflow/ops/model-config/mod';
+import { operation as SimulateCiemssOp } from '@/components/workflow/ops/simulate-ciemss/mod';
+import { operation as CompareDatasetOp } from '@/components/workflow/ops/compare-datasets/mod';
+import { operation as InterventionOp } from '@/components/workflow/ops/intervention-policy/mod';
+import { OperatorNodeSize } from '@/services/workflow';
+import { createModelConfiguration, getModelConfigurationById, getParameter } from '@/services/model-configurations';
+import _ from 'lodash';
+import { ChartSetting, ChartSettingType } from '@/types/common';
+import { updateChartSettingsBySelectedVariables } from '@/services/chart-settings';
+import { AssetType, ParameterSemantic } from '@/types/Types';
+import { DistributionType } from '@/services/distribution';
+import { calculateUncertaintyRange } from '@/utils/math';
+import { getInterventionPolicyById } from '@/services/intervention-policy';
+import { useProjects } from '@/composables/project';
+
+export interface HorizonScanningParameter {
+	id: string;
+	low: number;
+	high: number;
+}
+export class HorizonScanningScenario extends BaseScenario {
+	public static templateId = 'horizon-scanning';
```

Comment:
A short description of what is a horizon-scanning-scenario is here would be appreciated!

Reply:
added a blurb here